### PR TITLE
Add binning 

### PR DESCRIPTION
**Pull request by thomasbtf**

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -2,3 +2,10 @@
 # In case of sample based data, it should be complemented by a samples.tsv file that contains
 # one row per sample. It can be parsed easily via pandas.
 pepfile: config/pep/config.yaml
+
+# minimal confidence of the LSTM network of the detected word
+min-confidence: 0.6
+
+# maximal number number of single-character edits that are required to transform 
+# one string (the word on the image) into another (the word in the personal data)
+max-distance: 2
```
Comment (simakro) [inline config/config.yaml]: add newline at eof

Comment (thomasbtf) [reply config/config.yaml]: done in 0b5e45b. But why is this important?

File: workflow/rules/common.smk [hunk 2]
```diff
@@ -58,3 +58,48 @@ def get_personal_data(wildcards):
...
+            paths = pd.read_csv(f, sep="\n", header=None, squeeze=True)
+
+    elif case == "lots_personal_data_found":
+        pattern = "results/{id}/to-check/lots_personal_data_found/{{img}}".format(
+            id=wildcards.id
+        )
+        with checkpoints.create_paths_for_manually_checking.get(
+            id=wildcards.id
+        ).output.no_personal_data_found.open() as f:
+            paths = pd.read_csv(f, sep="\n", header=None, squeeze=True)
```
Comment (simakro) [inline workflow/rules/common.smk]:
```suggestion
    elif case == "lots_personal_data_found":
        pattern = "results/{id}/to-check/lots_personal_data_found/{{img}}".format(
            id=wildcards.id
        )
        with checkpoints.create_paths_for_manually_checking.get(
            id=wildcards.id
        ).output.lots_personal_data_found.open() as f:
            paths = pd.read_csv(f, sep="\n", header=None, squeeze=True)
```

File: workflow/scripts/create-paths-for-manually-checking.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    )
+
+
+def name_not_entirely_found(summary_df: pd.DataFrame, out_path: str):
+    save_df(
+        summary_df[summary_df["name_family"] != summary_df["name_first_0"]][
+            ["processed img"]
+        ],
+        out_path,
+    )
```
Comment (simakro) [inline workflow/scripts/create-paths-for-manually-checking.py]: Not sure if this will work (particularly with the names)

**Review by simakro — CHANGES_REQUESTED** [review]

File: workflow/scripts/create-paths-for-manually-checking.py (hunk 3 above)
Comment (thomasbtf) [reply workflow/scripts/create-paths-for-manually-checking.py]: So far the results of this query seem okay. Of course, there are some false positives (e.g. the last name occurs in a text), but also false negatives are detected.

**Review by simakro — APPROVED** [review]

**Pull request merged**